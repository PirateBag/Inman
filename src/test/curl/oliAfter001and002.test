Generate a report of all order line items.
This report does not yet support display children of a master PO or Master MO.

Setup:
Orders for W-001 and W-002 have been created.
Expected output:
Empty report but no failures.

=== JSON ===
{ "idToSearchFor" : "-1" }