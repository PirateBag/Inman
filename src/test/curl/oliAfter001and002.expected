{
  "timestamp" : "2025-06-17T17:05:10.746+00:00",
  "status" : 404,
  "error" : "Not Found",
  "path" : "/oli/showall"
}